admin-dev/themes/new-theme/public/theme.rtlfix

```diff
@@ -183,6 +183,6 @@ ul.category-tree .more > .radio:before,
 }
 
 /* Fix dropdown overflow inside grid table */
-.grid-table {
-  overflow-x: auto;
+.table-responsive {
+  overflow-y: hidden;
```

Comment:
Can you check the dropdown of the last line inside a table? I think it can cause some bugs!

Reply:
https://watch.screencastify.com/v/dGcMMM4lxpPBBGfipHIF

Reply:
Mobile test:
https://user-images.githubusercontent.com/85633460/145776442-3218ad2b-adfe-4c5f-b4c3-d421e966e4c9.mp4